cmd/beaker/workspace.go

```diff
@@ -468,7 +469,8 @@ func newWorkspaceUnarchiveCommand() *cobra.Command {
 		Short: "Unarchive a workspace",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := beaker.Workspace(args[0]).SetArchived(ctx, false); err != nil {
+			falseVal := false
+			if _, err := beaker.Workspace(args[0]).Patch(ctx, api.WorkspacePatch{Archive: &falseVal}); err != nil {
```

Comment:
We have a helper for this.  Also nit-pick: long lines don't format well in GitHub.

```suggestion
			if _, err := beaker.Workspace(args[0]).Patch(ctx, api.WorkspacePatch{
				Archive: api.BoolPtr(false),
			}); err != nil {
```